code/style.css

```diff
@@ -1,5 +1,191 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
-
-body {
-  background-color: peachpuff;
-}
+
+/*** Body ***/
+
+body {
+    background-color: whitesmoke;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 15px;
+}
+
+/*** Header ***/
+
+header {
+    background-color: white;
+    height: 80px;
+    display: flex;
+    position: sticky;
+    top: 0;
+    z-index: 100;
+    align-items: center;
+    overflow: hidden;
+    padding: 0 16px;
+    opacity: 0.95   ;
+}
+
+header img {
+    width: 180px;
+}
+
+.hamburger-menu {
+    display: flex;
+    width: 25px;
+    height: 15px;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-right: 24px;
+}
+
+.hamburger-menu div {
+    background: black;
+    width: 30px;
+    height: 1px;
+    border-radius: 4px;
+    padding: 1px;
+}
+
+nav a {
+    color: black;
+    text-decoration: none;
+    margin-left: 16px;
+    font-weight: bold;
+}
+
+/*** Main Content ***/
+
+.welcome {
+    display: flex;
+    padding-top: 10px;
+}
+
+.welcome img {
+    width: 100%;
+}
+
+.welcome-text {
+    color: rgb(0, 40, 165);
+    text-align: center;
+    font-size: large;
+}
+
+img {
+	display: block;
+	border: 0;
+	width: 100%;
+	height: auto;
+}
+
+.card {
+	background: white;
+    margin-bottom: 20px;	
+}
+
+.card a {
+	color: black;
+	text-decoration: none;
+}
+
+#link-orange {
+	color: rgb(218, 84, 46);
+	text-decoration: none;
+    font-weight: bold;
+}
+
+
+.card-content {
+	padding: 1.4em;
+}
+
+.card-content h2 {
+	margin-top: 0;
+	margin-bottom: .5em;
+	font-weight: bold;
+}
+
+.cards {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+ }
+
+.card {
+    flex: 0 1 calc(25% - 1em);
+}
+
+  /*** Footer ***/
+
+  footer {  
+    display: flex;
+    background-color: rgb(0, 94, 168);

```

Comment:
The most common way to declare colors is to use hex-values, ex. #ffffff or #000000. Rgb is good, but mainly used if you want to add some opacity as you can work with rgba (red, green, blue, alpha) to add a value for opacity/alpha. There are many converters online if you wish to convert your selector color to hex. 😊